dspace/config/ldn/request-ingest

```diff
@@ -10,11 +10,12 @@
 ##             {7} the url to the primary bitstream or the first bitstream in the ORIGINAL bundle if there is no primary bitstream. The url is 'dspace.ui.url'/bitstreams/:uuid/download
 ##             {8} the bitstream  MimeType or get User Format MimeType if getFormat is 'Unknown'
 ##             {9} id of the created LDNMessage
+##            {14} config 'dspace.ui.url'
```

Comment:
same as for the other file (referring to parameter number 14)

> Is there a reason why a new parameter was added even though from the description above I see that the parameter is equal to parameter 0? I think it's probably a typo in the description of this file and the parameter is not always intended to be equal to parameter 0. In this case a better description of the parameter might be needed

Reply:
Removed as not needed